config-template.js

```diff
@@ -0,0 +1,79 @@
+/**
+ * Copy this file to config.js and fill it in with your own information.
+ */
+module.exports = {
```

Comment:
It's weird, I never saw that before (Sent to GitHub as the User-Agent as required by their API.
         * See https://github.com/settings/developers for more info.) 

I think i missed it or something. But with that in there I think it's good! 🚢  it!